test/controllers/tests_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class TestsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```

Comment:
If youre not going to use this delete it

Reply:
Fixed!